server.js

```diff
@@ -1,29 +1,132 @@
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({
+    "World of Instant Ramen": "Because Ramen safes your busy life",
+      "Routes":[{
+        "/ramen": "get list of all ramen",
+        "/ramen/id/:id": "get ramen's id and return a single result",
+        "/ramen/country/:country": "get list of all ramen from specific country",
+        "/ramen/brand/:brand": "get list of all ramen with specific brand",
+        "/ramen/style/:style": "get list of all ramen with specific type of serving",
+        "/ramen/best": "get list of ramen from the highest rating to lowest rating"
+      }],
+      "Queries":[{
+        "/ramen/country/:country?bad=true": "get list of ramen in a specific country that has less than 3 stars",
+        "/ramen/country/:country?good=true": "get list of ramen in a specific country that has more than 4 stars"
+      }]
+  });
+});
+
+
+// Route to all ramen data
+app.get("/ramen", (req, res) => {
+  let ramenData = data
+  res.status(200).json(ramenData)
+})
+
+// Route to ramen's id and return a single result
+// example route: /ramen/id/2502
+app.get("/ramen/id/:id", (req, res) => {
+  const id = req.params.id
+  let selectedId = data.find((item) => item.Id === +id)
+  if (!selectedId){
+    res.status(404).send({
+      data: "ID not found",
+      success: false 
+    })
+  } else {
+    res.status(200).json({
+      data: selectedId,
+    success: true
+  });
+  }
+});
+
+// Route to ramen based on distribution between countries
+// example route: /ramen/country/indonesia
+// example route to good ramen: ramen/country/south%20korea?good=true
+app.get("/ramen/country/:country", (req, res) => {
+  const country = req.params.country
+  const { bad, good } = req.query
+  let selectedCountry = data.filter((item) => item.Country.toLowerCase() === country.toLowerCase())
+  if(bad){
+    selectedCountry = selectedCountry.filter((item) => item.Stars < 3)
+  }
+  if(good){
+    selectedCountry = selectedCountry.filter((item) => item.Stars > 4)
+  }
+
+  res.status(200).json(selectedCountry);
 });
 
+// Route to ramen based on the brand name
+// example route: /ramen/brand/indomie
+app.get("/ramen/brand/:brand", (req, res) => {
+  const brand = req.params.brand
+  let selectedBrand = data.find((item) => item.Brand.toLowerCase() === brand.toLowerCase())
+  
+  if (!selectedBrand){
+    res.status(404).send({
+      data: "Brand not found",
+      success: false 
+    })
+  } else {
+    res.status(200).json({
+      data: selectedBrand,
+      success: true
+    });
+  } 
+});
```

Comment:
https://project-express-api-5dpplzryzq-lz.a.run.app/ramen/brand/nissin  only displays one item not a list.
I think it's because you used find()-method and should use the filter()-method.
(see the ramen/country/:country endpoint which uses filter() and returns a list)
from MDN Web docs mozilla : 
The find() method is an iterative method. It calls a provided callbackFn function once for each element in an array in ascending-index order, until callbackFn returns a truthy value. find() then returns that element and stops iterating through the array.

Reply:
Thank you Nina for this comment, I used filter before but somehow I changed it to find and I don't know why honestly haha but thanks for pointing it out!